src/conversationLogger.js

```diff
@@ -21,13 +23,13 @@ class ConversationLogger {
         json: true
       });
     }
-    this.dashbotClient = this.options.dashBotKey ? dashbot(this.options.dashBotKey).facebook : false;
+    this.dashbotClient = this.options.dashBotKey ? dashbot(this.options.dashBotKey).facebook : null;
```

Comment:
I changed this from `false` to `null` to follow Flow's `?Object` notation. It still evaluates as falsey and is semantically the same or better.